content/en/docs/setup/install/multiple-controlplanes/index.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Install Multiple Istio Control Planes in a Single Cluster
+description: Install multiple Istio control planes in a single cluster using revisions and discoverySelectors.
+weight: 55
+keywords: [multiple,control,istiod,local]
+owner: istio/wg-environments-maintainers
+test: yes
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+This guide walks you through the process of installing multiple Istio control planes within a single cluster and
+then a way to scope workloads to specific control planes. For this guide, [the deployment model](/docs/ops/deployment/deployment-models/#control-plane-models)
```

Comment:
Referencing the deployment model doc here is confusing, since it doesn't mention anything about the model you're using, i.e., you have one cluster with two meshes inside it. The control plane models in the doc all have one mesh (blue dashed-line box) at the outside, with the clusters (gold solid-line boxes) inside. You're describing a deployment that has the gold solid-line box at the outside with two single control plane meshes (blue boxes) inside of it. I would suggest adding such a diagram in the deployment doc, or don't reference it at all.

Reply:
Removed